### ISSUE TITLE
Screen readers unable to read contents in editor

### ISSUE THREAD

**Issue by zersiax** [Zettlr/Zettlr#5183]
### Description

When opening a new file in Zettlr, and using a screen reader to try to review your input, it will rapdily become apparent that screen readers are unable to perceive this content.
This almost certainly happens due to the specific editor component used, my guess would be CodeMirror on a version lower than 6, where this problem has (at least to a point) been resolved.
Please consider bumping the CodeMirror version, allowing for an alternative editor (Monaco would be best), or figuring out some other solution for the fact that people using a screen reader are currently unable to edit any kind of text whatsoever within this app, something that I would consider one of its core functions :)
Disclaimer: screen reader user with a developer background.

### Reproducing

1. Wihth a screen reader ( Mac VoiceOver, NVDA for Windows, Orca for lInux etc.) running, open Zettlr.
2. Open a new file.
3. Write some text.
4. Use the left arrow key to go through your just entered text.
5. Observe that only " blank"  or nothing is spoken by the screen reader's text to speech output.

### Zettlr Version

Stable (most recent version)

### Specify version

This is more than likely a problem that has been there for the entirety of the app's existence

### Installation Method

None

### Your Platform

- [ ] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [ ] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

irrelevant, but Windows 11 23h2

### Additional Information

For technical context, what's happening here is that screen readers can only focus one thing at a time, and that what they focus needs to entirely be part of the DOM. The textarea that CodeMirror (and perhaps very similar other editor components) use to work is really just there to catch input, not to hold output, therefore the contents of the editor actually isn't in the textarea, it's usually in some kind of canvas behind the textarea. The only time the entire contents of the editor is in the screenreader-accessible textarea element is when someone hits ctrl+a to select all, as the OS-level clipboard suffers from the same problem and wouldn't otherwise be able to copy the code/text out of the editor.

**Comment by nathanlesage** [comment]
Sorry that your issue went unresponded-to for a year. I just tried it out and the current version works just fine with macOS's built-in screen reader. Just for the record: Back in 2024 we were already using CM6; so it was likely an intermittent issue that has solved itself in the meantime.
